pkgs/applications/networking/mailreaders/hey-mail/default.nix

```diff
@@ -0,0 +1,114 @@
+{ fetchurl, lib, stdenv, squashfsTools, makeWrapper, c-ares, gtk3-x11,
+glib, libevent, libdrm, libvpx, libxslt, libnotify,
+libappindicator-gtk2, libappindicator-gtk3, atk, mesa, cups, systemd,
+alsaLib, at-spi2-atk, at-spi2-core, gdk-pixbuf, gnome2, cairo, xorg,
+ffmpeg, http-parser, nss, nspr, dbus, expat }:
+
+let
+  deps = [
+    c-ares
+    gtk3-x11
+    glib
+    libevent
+    libdrm
+    libvpx
+    libxslt
+    libnotify
+    libappindicator-gtk2
+    libappindicator-gtk3
+    atk
+    mesa
+    cups
+    systemd
+    alsaLib
+    at-spi2-atk
+    at-spi2-core
+    gdk-pixbuf
+    gnome2.pango
+    cairo
+    xorg.libxcb
+    xorg.libX11
+    xorg.libXcomposite
+    xorg.libXcursor
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXi
+    xorg.libXrender
+    xorg.libXtst
+    xorg.libXrandr
+    ffmpeg
+    http-parser
+    nss
+    nspr
+    dbus
+    expat
+    stdenv.cc.cc
+  ];
+  rev = "10";
+
+in
+
+stdenv.mkDerivation rec {
+  pname = "hey-mail";
+  version = "1.1.0";
```

Comment:
Maybe the `rev` should be defined here, if it's sort of related to the `version`.

Reply:
Is it derived from the version?